core/src/parser/program/operator.rs

```diff
@@ -0,0 +1,32 @@
+pub use crate::parser::parser::*;
```

Comment:
This file probably isn't needed as it is only used three times, and the `Operator` already implements `is_binary`, `is_unary` and `is_eq`